client/state/current-user/selectors.js

```diff
@@ -44,6 +44,21 @@ export function getCurrentUserLocale( state ) {
 }
 
 /**
+ * Returns the isRTL value for the current user.
+ *
+ * @param  {Object} state  Global state tree
+ * @return {?Bool}  Current user is RTL
```

Comment:
We usually use the full type name `Boolean` instead of `Bool`, although there are probably several more cases like this throughout Calypso.

Also, nitpicking: may I suggest that you align the different sections of the `jsDoc`? See docs of other selectors in this file for an example.